modules/overview-of-kueue-resources.adoc

```diff
@@ -0,0 +1,136 @@
+  nodeLabels:
+    instance-type: spot
+  nodeTaints:
+  - effect: NoSchedule
+    key: spot
+    value: "true"
+  tolerations:
+  - key: "spot-taint"
+    operator: "Exists"
+    effect: "NoSchedule"
+
+----
+
+For more information about configuring resource flavors, see link:https://kueue.sigs.k8s.io/docs/concepts/resource_flavor/[Resource Flavor] in the Kueue documentation.
+
+
+== Cluster queue
+
+The Kueue `ClusterQueue` object manages a pool of cluster resources such as pods, CPU, memory, and accelerators. 
+A cluster can have multiple cluster queues, and each cluster queue can reference multiple resource flavors.
+
+The cluster administrator can configure a cluster queue to define the resource flavors that the queue manages, and assign a quota for each resource in each resource flavor.
+The cluster administrator can also configure usage limits and queueing strategies to apply fair sharing rules across multiple cluster queues in the cluster.
+ 
+The following example configures a cluster queue to assign a quota of 9 CPU, 36 GiB memory, 5 pods, and 5 NVIDIA GPUs.
+
+.Example cluster queue
+[source,bash]
+----
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ClusterQueue
+metadata:
+  name: "cluster-queue"
+spec:
+  namespaceSelector: {} # match all.
+  resourceGroups:
+  - coveredResources: ["cpu", "memory", "pods", "nvidia.com/gpu"]
+    flavors:
+    - name: "default-flavor"
+      resources:
+      - name: "cpu"
+        nominalQuota: 9
+      - name: "memory"
+        nominalQuota: 36Gi
+      - name: "pods"
+        nominalQuota: 5
+      - name: "nvidia.com/gpu"
+        nominalQuota: '5'
+----
+
+The cluster queue will start a distributed workload only if the total required resources are within these quota limits. 
+If the sum of the requests for a resource in a distributed workload is greater than the specified quota for that resource in the cluster queue, the cluster queue does not admit the distributed workload.
+
+
+For more information about configuring cluster queues, see link:https://kueue.sigs.k8s.io/docs/concepts/cluster_queue/[Cluster Queue] in the Kueue documentation.
+
+
+== Local queue
+
+The Kueue `LocalQueue` object groups closely related distributed workloads in a project.
+The cluster administrator can configure a local queue to specify the project name and the associated cluster queue.
+The local queue then grants access to the resources that the specified cluster queue manages.
+The cluster administrator can optionally define one local queue in a project as the default local queue for that project.
+
+When configuring a distributed workload, the user specifies the local queue name.
+If the cluster administrator configured a default local queue, the user can omit the local queue specification from the distributed workload code.
+
+Kueue allocates the resources for a distributed workload from the cluster queue for the local queue, if the total requested resources are within the quota limits specified in that cluster queue.
```

Comment:
I find this sentence a little confusing but am not sure how to phrase it better. It's the `from the cluster queue for the local queue part` in particular.

Reply:
@Fiona-Waters Thanks a million for your review :) 

I changed the problem phrase to "from the cluster queue that is associated with the local queue", is that any better?

The text is now as follows:
"Kueue allocates the resources for a distributed workload from the cluster queue that is associated with the local queue, if the total requested resources are within the quota limits specified in that cluster queue."

I could change it as follows if you think it's better, but I'm not sure that it's accurate?
"Kueue allocates the resources for a distributed workload from the local queue, if the total requested resources are within the quota limits specified in the associated cluster queue."

Reply:
`from the cluster queue that is associated with the local queue` is much better!